modules/tide_landing_page/js/hero_image_theme.js

```diff
@@ -0,0 +1,39 @@
+(function (Drupal) {
+  "use strict";
+
+  Drupal.behaviors.heroImageTheme = {
+    attach: function (context, settings) {
+      const imageTheme = document.querySelector(
+        'select[data-drupal-selector^="edit-field-landing-page-hero-theme"]'
+      );
+
+      if (imageTheme !== null) {
+        setImageTheme();
+
+        const headerStyles = document.querySelectorAll(
+          'input[data-drupal-selector^="edit-header-style-options"]'
+        );
+
+        if (headerStyles !== null) {
+          headerStyles.forEach((style) => {
+            style.addEventListener("change", () => {
+              setImageTheme();
```

Comment:
Instead of repeatedly defining an anonymous function for style.addEventListener("change", ...), you can directly pass the reference to the setImageTheme function.